src/scss/_main.scss

```diff
@@ -779,7 +779,7 @@ nav {
   justify-content: flex-start;
 
   padding-right: 1rem;
-  padding-left: 2.6rem;
+  padding-left: 1.6rem;
```

Comment:
moving steps and services `1rem` to the left, to account for vertical space lost by the tabs